enhancements/sdk-migrate-bundle.md

```diff
@@ -0,0 +1,379 @@
+---
+title: sdk-migrate-bundle
+authors:
+  - "@jmrodri"
+reviewers:
+  - "@estroz"
+  - "@ecordell"
+  - "@rashmigottipati"
+approvers:
+  - "@estroz"
+  - "@rashmigottipati"
+creation-date: 2021-02-08
+last-updated: 2021-02-08
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# Migrate Bundle
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [X] Design details are appropriately documented from clear requirements
+- [X] Test plan is defined
+- [X] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions [optional]
+
+1. The bundles we build today are done via `docker` from the project's
+   `Makefile` where users can adapt their image builder to whatever they want.
+   Do we want to allow this from the CLI?
+   - if we do build images, we should add image builder flag
+1. What will happen to the existing command in SDK for deploying an operator
+   packagemanifests using OLM? Do we still want to support that and will it stay intact?
+   - we will deprecate the `run packagemanifests` command and remove in SDK 2.0
+
+## Summary
+
+Add a subcommand to the Operator SDK CLI that will allow users to migrate
+their operator's packagemanifest format to the new bundle format.
+
+## Motivation
+
+There are operators that exist that are still using the packagemanifest
+format. This subcommand will allow them to migrate these packagemanifests to
+bundles which is the preferred OLM packaging format.
+
+### Goals
+
+- Provide a subcommand to migrate one packagemanifest to a set of bundles.
+- Deprecate `packagemanifests` features:
+  - Deprecate `run packagemanifests`
+  - Deprecate `generate packagemanifests`
+
+### Non-Goals
+
+- Solve the world?
+- There will be no migration from bundles to packagemanifests
```

Comment:
not clear the above non-goals

Reply:
Solve the world is a joke :) and the second one is that we will only migrate ONE WAY. Packagemanifests -> bundles, but we will not go back the other way.